LibRangeCheck-3.0/LibRangeCheck-3.0.lua

```diff
@@ -524,8 +524,8 @@ end
 
 -- temporary stuff
 
-local pendingItemRequest
-local itemRequestTimeoutAt
+local pendingItemRequest = {}
```

Comment:
The minor version needs an increase. 

Otherwise this appears to work, although I'm not entirely sure why the item code is written in the way it is written.

Reply:
Bumped minor.

Can't say I understand why the item querying is done this way either. It is a good idea to spread it out across multiple frames like this though, since making too many `GetItemInfo` requests at a time can cause some of them to get lost and never return with `GET_ITEM_INFO_RECEIVED`. Anyway, it's been getting the job done so I don't think it needs to be rewritten. I only made minimal changes so it can keep track of multiple items at once.